CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ set(drishti_upload_init "${CMAKE_CURRENT_LIST_DIR}/drishti-upload/init.cmake")
 if(DRISHTI_USE_DRISHTI_UPLOAD)
   if(EXISTS "${drishti_upload_init}")
     # CI: Locked CI build configuration
+    include("cmake/HunterGate.cmake") # HUNTER_ENABLED
```

Comment:
Not needed, path to HunterGate will be added here:
* https://github.com/elucideye/drishti-upload/blob/3ee5b46cdd63c86cbbd8c534e5955fec9afb72f0/init.cmake#L36

Reply:
Something still needs to include it though.  If I remove that line and test with `DRISHTI_USE_DRISHTI_UPLOAD=YES` I get the following error:

```
HUNTER_RUN_UPLOAD: OFF
CMake Error at drishti-upload/init.cmake:38 (HunterGate):
  Unknown CMake command "HunterGate".
Call Stack (most recent call first):
  CMakeLists.txt:23 (include)
```

AFAIK, adding `cmake/Modules` to `CMAKE_MODULE_PATH` just allows us to shorten `include("cmake/HunterGate.cmake")` to `include("HunterGate.cmake")` after the `CMAKE_MODULE_PATH` has been updated.

Reply:
Going to merge for the fix w/ default `DRISHTI_USE_DRISHTI_UPLOAD`.  Let me know if you see a follow-up improvement.

Reply:
My bad, we need this include indeed.